src/recharge.iced

```diff
@@ -93,7 +132,16 @@ exports.Runner = class Runner extends Base
     change_offset = tx.addOutput @change_address, 1
     tx.sign 0, skey
 
-    fee = btcjs.networks.bitcoin.estimateFee(tx)
+    # Can change these settings...
+    btc_opts = {
```

Comment:
Should somehow read this out of the config file in the production (non-test) case.